tcms/rpc/tests/test_testcase.py

```diff
@@ -430,6 +493,27 @@ def test_fails_when_mandatory_fields_not_specified(self):
             )
 
 
+class TestRemovePermissions(APIPermissionsTestCase):
+    permission_label = 'testcases.delete_testcase'
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.case_1 = TestCaseFactory()
+        self.case_2 = TestCaseFactory()
+        self.case_3 = TestCaseFactory()
+
+        self.query = {'pk__in': [self.case_1.pk, self.case_2.pk, self.case_3.pk]}
+
+    def verify_api_with_permission(self):
+        len_deleted, _ = self.rpc_client.TestCase.remove(self.query)
```

Comment:
What's the second return value? IMO you can remove only 1 of these records and assert on the second value as well b/c API clients may be using that value.

Reply:
`TestCase.remove()` returns `TestCase.objects.filter(**query).delete()`. According to [Django's documentation](https://docs.djangoproject.com/en/3.1/topics/db/queries/#deleting-objects), `delete` method returns the number of objects deleted and a dictionary with the number of deletions per object type. 

I don't think the number of deletions per object will be helpful for our testing.